packages/universal-components/src/WithHover/__tests__/__snapshots__/withHover.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Hoverable - native should match snapshot 1`] = `
+<View
+  onMouseEnter={[Function]}
```

Comment:
When hover is available just on web, shouldn't this be just `<View>`?

Reply:
I don't think it matters. There is no mouse on mobile. So it should never be triggered.

Reply:
True, it shouldn't break anything.